packages/pkg-deps.json

```diff
@@ -6,51 +6,21 @@
          "dh-systemd"
       ],
       "renames" : {
-         "pyyaml" : {
-            "2" : "python-yaml",
-            "3" : "python3-yaml"
-         },
-         "contextlib2" : {
-            "2" : "python-contextlib2"
-         },
-         "pyserial" : {
-            "2" : "python-serial",
-            "3" : "python3-serial"
-         }
+         "pyyaml" : "python3-yaml",
+         "pyserial" : "python3-serial"
       },
       "requires" : [
          "procps"
       ]
    },
    "redhat" : {
       "build-requires" : [
-         "python-devel",
-         "python-setuptools"
+         "python3-devel"
       ],
       "renames" : {
-         "jinja2" : {
-            "3" : "python36-jinja2"
-         },
-         "jsonschema" : {
-            "3" : "python36-jsonschema"
-         },
-         "pyflakes" : {
-            "2" : "pyflakes",
-            "3" : "python36-pyflakes"
-         },
-         "pyyaml" : {
-            "2" : "PyYAML",
-            "3" : "python36-PyYAML"
-         },
-         "pyserial" : {
-            "2" : "pyserial"
-         },
-         "pytest": {
-             "3": "python36-pytest"
-         },
-         "requests" : {
-            "3" : "python36-requests"
-         }
+         "httpretty" : "",
+         "unittest2" : "",
+         "contextlib2" : ""
```

Comment:
These renames to the empty string are a trick to skip installing those dependencies on in systems in the "redhat" family, because the packages are missing. This was discussed in a post-standup with @raharper IIRC.

Reply:
* I think we still need to keep these dependencies for Fedora as I can install those packages there I believe so I think we need a separate centos distro represented as it is slightly different than fedora deps.

* The empty string you set here currently doesn't actually skip installing that specific dependency we would need a comparable change to read-dependencies: translate_pip_to_system_pkg to specifically ignore our empty string rename value otherwise it actually appends a standardized package name python3-<pip_name> if rename == "".

* Also, since the time of your original writing this PR. python3-httpretty looks like it has made it into CentOS Epel repo now, so we don't have to avoid trying to install it.

I've attached a diff to my full review that contains all of my suggestions for your thoughts.